src/lib.rs

```diff
@@ -225,7 +225,8 @@ pub use stable_deref_trait::{CloneStableDeref, StableDeref};
 mod arch;
 pub use arch::*;
 
-mod constants;
+pub mod constants;
+// For backwards compat.
```

Comment:
We could also do `#[doc(hidden)]` to ensure that it never shows up in rustdoc.

Reply:
It's only one line in the re-export section of the doc, which I think is fine since we are in fact re-exporting it. I'm not trying to discourage users from using the re-export if it's more convenient for them, I just don't want all the constants drowning out the rest of the main page of the docs. I'm not sure what's idiomatic for re-exports in rust though.